components/binary_sensor/mpr121.rst

```diff
@@ -0,0 +1,60 @@
+MPR121 Capacitive Touch Sensor
+==============================
+
+.. seo::
+    :description: Instructions for setting up MPR121 Capacitive Touch Sensor
+    :image: mpr121.png
+    :keywords: MPR121
+
+The ``mpr121`` sensor platform allows you to use your MPR121
+(`datasheet <https://cdn-learn.adafruit.com/downloads/pdf/adafruit-mpr121-12-key-capacitive-touch-sensor-breakout-tutorial.pdf>`__,
+`Adafruit`_) Capacitive Touch Sensor with esphomelib. The :ref:`I²C <i2c>` is
+required to be set up in your configuration for this sensor to work.
+
+.. figure:: images/mpr121.png
+    :align: center
+    :width: 50.0%
+
+    MPR121 Capacitive Touch Sensor.
+
+.. _Adafruit: https://learn.adafruit.com/adafruit-mpr121-12-key-capacitive-touch-sensor-breakout-tutorial/overview
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    mpr121:
+      id: mpr121_component
+      address: 0x5A
+
+    binary_sensor:
+      - platform: mpr121
+        id: touch_key0
+        channel: 0
+        name: "Touch Key 0"
+
+Configuration variables:
+------------------------
+
+The configuration is made up of two parts: The central component, and individual Binary sensors per channel.
+
+Base Configuration:
+
+- **address** (*Optional*, integer): The i2c address of the sensor. Defaults to ``0x5A``.
+- **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor.
+
+Binary Sensor Configuration:
+
+- **name** (**Optional**, string): The name for the binary sensor.
+- **id** (*Optional*, :ref:`config-id`): Manually specify the ID used for code generation.
+- **channel** (*Required*, integer): The channel number at the mpr121 the touchkey is connected to.
+- All other options from :ref:`Binary Sensor <config-binary_sensor>` and :ref:`MQTT Component <config-mqtt-component>`.
```

Comment:
This line has changed since the 1.11 update, can you copy the new way this line looks from the other binary sensor files?

Reply:
We need to remove the two parts `Base Conficuration` and `Binary Sensor Configuration` and only have the `Configuration variables`?

Reply:
No I mean like this (see for example gpio binary sensor docs).

```suggestion
- All other options from :ref:`Binary Sensor <config-binary_sensor>`.
```